airflow/docker.go

```diff
@@ -211,6 +212,22 @@ func Start(airflowHome string, envFile string) error {
 		return errors.Wrap(err, "error retrieving working directory")
 	}
 
+	// parse dockerfile
+	cmd, err := docker.ParseFile(filepath.Join(airflowHome, dockerfile))
+	if err != nil {
+		return errors.Wrapf(err, "failed to parse dockerfile: %s", filepath.Join(airflowHome, "Dockerfile"))
+	}
+
+	_, airflowTag := docker.GetImageTagFromParsedFile(cmd)
+
+	semVer, err := semver.NewVersion(airflowTag)
+
+	if err != nil {
+		return errors.Wrapf(err, "failed to parse dockerfile Airflow tag: %s", airflowTag)
+	}
+
+	airflowDockerVersion := semVer.Major()
```

Comment:
@adam2k let's create new function `airflowVersionFromDockerFile`

Reply:
`airflowDockerVersion = airflowVersionFromDockerFile(dockerfile)`

Reply:
Do you think this should go in the `airflow/docker.go` file or in the `airflow/airflow.go` file?  I could see it in either place.

Reply:
just create private function somewhere around `airflow/docker.go`.

Reply:
added and updated the tests.